### ISSUE TITLE
vtk bottle download requires HTTP Basic login

### ISSUE THREAD

**Issue by dakcarto** [Homebrew/homebrew-science#4660]
### Indicate troubleshooting steps taken

- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
- [x] Checked that a similar issue hasn't already been filed?
- [x] Looked up the software bug tracker if reporting a build error?
- [x] If you're seeing permission errors tried running `sudo chown -R $(whoami) $(brew --prefix)`?

#### Brief summary of issue:

The following bottle URL fails with "curl: (22) The requested URL returned error: 401 Unauthorized":
https://homebrew.bintray.com/bottles-science/vtk-7.1.0.el_capitan.bottle.tar.gz

#### The complete command that you issued:

`brew upgrade vtk`

#### Gist with output of `brew doctor`:

```
Warning: Your Xcode (7.3.1) is outdated.
Please update to Xcode 8.1 (or delete it).
Xcode can be updated from the App Store.
```

#### Output of `brew config`:

```
HOMEBREW_VERSION: 1.1.2
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 0f529dae1043639058130092cf35e68d18907cb8
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: a4709f466b7d8ef16ef831a449188bf5ddaf206a
Core tap last commit: 3 days ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: octa-core 64-bit haswell
Homebrew Ruby: 2.0.0-p648
Clang: 7.3 build 703
Git: 2.10.2 => /usr/local/bin/git
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.12_2/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 1.8.0_74, 1.7.0_79
macOS: 10.11.6-x86_64
Xcode: 7.3.1
CLT: 7.3.1.0.1.1461711523
X11: 2.7.9 => /opt/X11
```

**Comment by jonchang** [comment]
Thanks for letting us know. I manually published the VTK 7.1 bottles so it should work now.

**Comment by dakcarto** [comment]
Confirmed. Thanks!
